pppt.gemspec

```diff
@@ -2,7 +2,7 @@
 
 SEQUEL_GEMSPEC = Gem::Specification.new do |s|
   s.name = 'pppt'
-  s.version = '0.4.1'
+  s.version = '0.5.0'
```

Comment:
Since it's a bug fix I think the convention should be to increment the last digit. So it would be '0.4.2'.
But since its only for our use, it's not a big deal.